File: app/src/main/java/com/github/florent37/expectanim/sample/ExpectAnimMainActivity.java
package com.github.florent37.expectanim.sample;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class ExpectAnimMainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.expectanim_activity_main);
        ButterKnife.bind(this);
    }

    @OnClick(R.id.sample)
    public void onSampleClicked() {
        startActivity(new Intent(this, ExpectAnimSampleActivity.class));
    }

    @OnClick(R.id.scroll)
    public void onScrollClicked() {
        startActivity(new Intent(this, ExpectAnimScrollActivity.class));
    }

    @OnClick(R.id.rotation)
    public void onRotationClicked() {
        startActivity(new Intent(this, ExpectAnimRotationActivity.class));
    }

    @OnClick(R.id.flip)
    public void onFlipClicked() {
        startActivity(new Intent(this, ExpectAnimFlipActivity.class));
    }

    @OnClick(R.id.setnow)
    public void onSetNowClicked() {
        startActivity(new Intent(this, ExpectAnimSetNowActivity.class));
    }

    @OnClick(R.id.visible)
    public void onVisibleClicked() {
        startActivity(new Intent(this, ExpectAnimAlphaActivity.class));
    }

    @OnClick(R.id.concat)
    public void onConcatClicked() {
        startActivity(new Intent(this, ExpectAnimConcatActivity.class));
    }

}

File: app/src/main/java/com/github/florent37/expectanim/sample/ExpectAnimMainActivity.kt
package com.github.florent37.expectanim.sample

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.expectanim_activity_main.*


class ExpectAnimMainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.expectanim_activity_main)

        sample.setOnClickListener {
            openScreen(ExpectAnimSampleActivity::class.java)
        }

        scroll.setOnClickListener {
            openScreen(ExpectAnimScrollActivity::class.java)
        }

        rotation.setOnClickListener {
            openScreen(ExpectAnimRotationActivity::class.java)
        }

        flip.setOnClickListener {
            openScreen(ExpectAnimFlipActivity::class.java)
        }

        setnow.setOnClickListener {
            openScreen(ExpectAnimSetNowActivity::class.java)
        }

        visible.setOnClickListener {
            openScreen(ExpectAnimAlphaActivity::class.java)
        }

        concat.setOnClickListener {
            openScreen(ExpectAnimConcatActivity::class.java)
        }
    }

    private fun openScreen(screenClass: Class<*>) {
        startActivity(Intent(this@ExpectAnimMainActivity, screenClass))
    }

}
